src/App.js

```diff
@@ -1,33 +1,32 @@
 import React, { Component } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
+import { Search } from './components/Search';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  saveInputValue = value => this.setState({ query: value });
 
   render() {
+    const { query } = this.state;
+
+    const filterMovies = moviesFromServer.filter(moviesSearch);
+
+    function moviesSearch({ title, description }) {
+      return title.toLowerCase().includes(query.toLowerCase())
+        || description.toLowerCase().includes(query.toLowerCase())
+        || query === '';
```

Comment:
```suggestion
      return (title + description).toLowerCase().includes(query.toLowerCase());
```
Этого должно хватить